hassio_aw2m/config.yaml

```diff
@@ -0,0 +1,24 @@
+name: AmbientWeather2MQTT
+version: "1.0.0"
+slug: hassio_aw2m
+description: This package listens for local data from Ambient Weather stations (such as the WS-2902C) and converts the incoming data to MQTT events.
+url: "https://github.com/neilenns/ambientweather2mqtt"
+startup: application
+boot: auto
+ports:
+  4500/tcp: 8132
+# TODO need to make the options respected as env variables
+options:
+  MAC_ADDRESS: ""
+  MQTT_SERVER: ""
+  MQTT_USERNAME: ""
+  MQTT_PASSWORD: ""
+  TZ: "America/Chicago"
+environment:
+  PORT: 8132
+services:
+  - "mqtt:need"
+arch:
+  - amd64
+  - armv7
+  - aarch64
```

Comment:
If possible, add all supported architectures. The complete list is:

armhf, armv7, aarch64, amd64, i386